modules/cnf-performing-platform-verification-latency-tests.adoc

```diff
@@ -0,0 +1,756 @@
+// Module included in the following assemblies:
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-performing-platform-verification-latency-tests_{context}"]
+= Performing latency tests for platform verification
+
+You can use the link:https://quay.io/repository/openshift-kni/cnf-tests?tag=latest&tab=tags[Cloud-native Network Functions (CNF) tests image] to run latency tests on a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed. The latency tests allows you to validate node tuning for your workload.
```

Comment:
quay should not be used in the documentation for images of red-hat products. please use the official one instead.

Reply:
Updated.